src/main/java/gregtech/common/metatileentities/multi/electric/generator/LargeCombustionEngineWorkableHandler.java

```diff
@@ -34,6 +38,7 @@ public FluidStack getFuelStack() {
     protected boolean checkRecipe(FuelRecipe recipe) {
         FluidStack lubricantStack = Materials.Lubricant.getFluid(2);
         FluidStack drainStack = fluidTank.get().drain(lubricantStack, false);
+        //TODO, shouldn't this be &&?
```

Comment:
I think this should be &&.

Reply:
I will leave this one alone for now, as it is outside of the scope of the PR and was just something I noticed when making changes in the class. Plus it requires a bit more thinking about.